.gitignore

```diff
@@ -17,3 +17,6 @@
 /tmp
 .DS_Store
 *~
+
+# Ignore application configuration
+/config/application.yml
```

Comment:
If I understand this correctly, this YAML file is created/used by the figaro gem. I've never used figaro before but the description says its a Heroku-friendly config file. I'm assuming we will need some sort of application file for the heroku (production) version